### PR TITLE
Return the current location from setGeoLocation

### PR DESCRIPTION
**Pull request by mykola-mokhnach**
Addresses https://github.com/appium/appium/issues/7400

### CODE REVIEW

File: test/unit/commands/network-specs.js [hunk 1]
```diff
@@ -145,8 +145,16 @@ describe('Network', function () {
   describe('setGeoLocation', function () {
     it('should set location', async function () {
```
Comment (imurchie) [inline test/unit/commands/network-specs.js]: This is really not test that it "should set location" so much as that it returns the geo location.

Comment (mykola-mokhnach) [reply test/unit/commands/network-specs.js]: which name would be better?

Comment (imurchie) [reply test/unit/commands/network-specs.js]: Something like?
```js
it('should return location in use after setting', async function () {
```

Comment (mykola-mokhnach) [reply test/unit/commands/network-specs.js]: fixed

**Pull request merged**